src/client/elm/Explorer/Stubs.elm

```diff
@@ -1054,3 +1054,585 @@ getBlockSummaryResponseStub =
   ]
 }
 """
+
+
+getBlockSummaryLongMemoStub =
```

Comment:
I'm OK with this, but would appreciate it if there was a bit of documentation. The variable name is indicative, but only now that we remember this, so a couple of sentences would be appreciated.

Reply:
I added a comment, but i'm fine with removing the whole thing too if thats preferred.

Reply:
Looks good.